crates/typst/src/eval/methods.rs

```diff
@@ -152,6 +179,19 @@ pub fn call(
             _ => return missing(),
         },
 
+        Value::Length(length) => match method {
+            "cm" => length.abs.to_cm().into_value(),
```

Comment:
I think `pt` should be here to and then we drop the `abs` field. There is no reason why `pt` should be special. 

It could be confusing that this just ignores the `em` part, but not much we can do about it (without future lazy magic get rules). But probably `em` should then also be a method here and we drop all fields?

Reply:
We could also _keep_ the `abs` field, add `pt()` here and have these functions throw if `em` is not `0`. then you can do `length.abs.cm()`, but `2em.cm()` fails instead of giving  `0.0`.

Reply:
> We could also _keep_ the `abs` field, add `pt()` here and have these functions throw if `em` is not `0`. then you can do `length.abs.cm()`, but `2em.cm()` fails instead of giving `0.0`.

This sounds more appropriate to me. Perhaps we should add this as a hint as well

Reply:
Done. Make sure to check the error/hint messages and the docs for any changes you might deem necessary.